README.md

```
@@ -1,273 +1,49 @@
- Privated Packages of RealThread  --->
-        [*] rdb:RT-Thread Debug Bridge                                                         
-               rdb version (v1.0.1)  --->
-```
+### 初始化流程
 
-rdb package 版本要选择  v1.0.1，并且下载到本地之后要手动将分支切换到远端最新代码然后重新生成工程
+RT_Insight 需在 RT-Tunnel 初始化后启动，推荐初始化顺序：
 
-首先在 env 输入命令 `rdb shell` 进入rdb shell 模式，然后输入 rti_start_record 开始录制，输入 rti_stop_record 结束录制
+1. **初始化 RT-Tunnel**：确保 `RT_Tunnel_Init` 先执行（通常自动初始化）；
+2. **初始化 RT_Insight **：调用 `RT_Vterm_Init` 分配隧道并初始化insight功能；
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)